src/actions/LoginAction.tsx

```diff
@@ -61,7 +61,8 @@ export const launchPasswordRecovery = (recoveryKey: string) => async (
 
       const usernameError = asMaybeUsernameError(error)
       if (usernameError != null) {
-        return s.strings.recovery_by_username_error
+        showError(s.strings.recovery_by_username_error)
```

Comment:
This makes it so the fn only returns booleans now but I'm fine since we plan to update in the future